tests/test_measures.py

```diff
@@ -462,6 +462,71 @@ def circuit2():
         assert np.allclose(circuit1(), circuit2(), atol=tol)
 
 
+class TestSample:
+    """Tests that samples are properly calculated."""
+
+    @pytest.mark.parametrize("C", [np.complex64, np.complex128])
+    def test_sample_dimensions(self, C):
+
+        # Explicitly resetting is necessary as the internal
+        # state is set to None in __init__ and only properly
+        # initialized during reset
+        dev = qml.device("lightning.qubit", wires=2, shots=1000)
```

Comment:
You may want to add a fixture for dev as follows and use it for all these tests.
```Python
    @pytest.fixture
    def dev(self):
        return qml.device("lightning.qubit", wires=2, shots=1000)
```

Reply:
nice will do